paper.md

```diff
@@ -211,19 +211,19 @@ einer *Era Number*, einem in Sekunden gemessenen *Era Offset* und einem Bruch
 besteht. Die *Era Number* bezeichnet den Bereich, in dem der 32-bit Offset nicht
 überläuft. Momentan sind wir in Era 0; ab dem 08. Februar 2036 werden wir in
 Era 1 sein. Das im Protokoll verwendete *Timestamp*-Format hat einen 32-bit
-Sekundenzähler und einen Bruch; das *Short*-Format ist ähnlich, hat aber nur
+Sekundenzähler und einen Bruch[für Millisekunden? nicht ganz klar]; das *Short*-Format ist ähnlich, hat aber nur
 16 Bit Präzision.
 
 TCP kann hier nicht verwendet werden, weil es verlorene Pakete wieder
 überträgt und dadurch die Zeitstempel in diesen verfälscht [citation needed],
 deswegen wird UDP auf Port 123 verwendet. NTP verwendet konventionelle binäre
 Pakete mit einem Header und einem aus vier Timestamps bestehenden Payload. Ein
-invalider Wert im Header, Stratum 0, initiiert ein *Kiss-o'-Death*-Paket, mit
+invalider Wert im Header, Stratum 0, initiiert ein *Kiss-o'-Death*-Paket[warum der Name], mit
```

Comment:
The protocol creators are hackers and chose to use this well-established term for control packets.